README.md

```diff
@@ -235,3 +243,35 @@ TAG=gcr.io/${PROJECT}/dicom-export-adapter
 gradle dockerBuildImage -Pdocker_tag=${TAG}
 docker push ${TAG}
 ```
+
+### Running Docker Images locally
+
+Unlike gce/gke, simple docker run doesn't automatically handle google authentication credentials.
+Solution is to mount local credentials file as a volume:
+
+```shell
+-v /path/to/google_credentials.json:/credentials -e GOOGLE_APPLICATION_CREDENTIALS='/credentials'
+```
+
+Import adapter also needs AET dictionary json to support C-MOVE and Storage Commitment service:
+
+```shell
+-v /path/to/HealthcareProjects/AETs.json:/AETs
```

Comment:
I think we could remove HealthcareProjects from the path and just have path to AETs.json similar to credentials json?

Reply:
Yes, missed it. Fixed in wiki.